posts/GSoC 2019.md

```diff
@@ -0,0 +1,64 @@
+---
+title: GSoC 2019 PyCanChemAndYouCan2
+date: 2019-06-07 12:33:00 UTC-16:33
+slug: GSoC 2019
+tags: GSoC 2019
+description:  2019 Google Summer of Code Cnatera Project Blog 
+type: text
+author: Chao Xu
+
```

Comment:
```suggestion
```

Reply:
Hi Bryan, thanks for your comments. I just revised my blog, especially the  "How can I achieve" part. I will meet with Rechard and make checkpoints next week.